serendipity_plugin_shoutbox/serendipity_plugin_shoutbox.php

```diff
@@ -227,7 +217,7 @@ function generate_content(&$title)
                 serendipity_plugin_api::hook_event('frontend_display', $entry);
                 $entry['comment'] = wordwrap($entry['comment'], $wordwrap, "\n", 1);
 
-                echo '<div class="serendipity_shoutbox_date">' . (function_exists('serendipity_specialchars') ? serendipity_specialchars(serendipity_strftime($dateformat, $row['stamp'])) : htmlspecialchars(serendipity_strftime($dateformat, $row['stamp']), ENT_COMPAT, LANG_CHARSET)) . '</div>' . "\n"
+                echo '<div class="serendipity_shoutbox_date">' . serendipity_specialchars(serendipity_strftime($dateformat, $row['stamp'])) . '</div>' . "\n"
```

Comment:
With this change (and the unguarded function call above) the minimal serendipity function should be set to when that function was introduced, which was `2.0`.